Rubberduck.Core/Properties/Settings.settings

```diff
@@ -231,7 +231,7 @@
               &lt;CodeInspection Name="EmptyModuleInspection" Severity="Hint" InspectionType="MaintainabilityAndReadabilityIssues" /&gt;
               &lt;CodeInspection Name="LineLabelNotUsedInspection" Severity="Warning" InspectionType="CodeQualityIssues" /&gt;
               &lt;CodeInspection Name="IntegerDataTypeInspection" Severity="Hint" InspectionType="CodeQualityIssues" /&gt;
-              &lt;CodeInspection Name="ShadowedDeclarationInspection" Severity="DoNotShow" InspectionType="CodeQualityIssues" /&gt;
+              &lt;CodeInspection Name="ShadowedDeclarationInspection" Severity="Warning" InspectionType="CodeQualityIssues" /&gt;
```

Comment:
Hmm, I would expect `App.config` to mirror that change.

Reply:
Ok, so I just wrote a lengthy reply explaining in how many ways I don't understand what you're saying, and justifying what I had to make sure it all worked. Just before posting, it dawned on me that I might have missed one test: Running RD without a config file. Now I see what you mean.

That unfortunately doesn't mean I understand what exact `App.config` you're referring to (there's none in the `.Resources` namespace). And I don't get the how and why. In my very simple mental model of RD's inner workings, it seems like we'd like to have such things stored in only one place.

And if you now tell me to go and read a book about C# software architecture and XAML, I guess you're probably right. ;)

Reply:
Ah, sorry - that would be under `Rubberduck.Core`: https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Core/app.config

Reply:
After building, `app.config` should mirror the change.